files/class-api-cache.php

```diff
@@ -40,14 +45,17 @@ public static function get_instance() {
 	}
 
 	public function clear_tmp_files() {
-		if ( empty( $this->files ) ) {
+		if ( empty( $this->files ) && empty( $this->file_stats ) ) {
 			return;
 		}
 
 		foreach( $this->files as $name => $path ) {
 			unlink( $path );
 			unset( $this->files[ $name ] );
 		}
+
+    // empty file stats cache
+    $this->file_stats = [];
```

Comment:
Looks like the whitespace on these lines got a little funky.

Reply:
Actually, I think most of the new lines of code have funky whitespace.

Reply:
Ah looks like vim isn't remembering my `noexpandtab` setting for some reason 😐

Fixed in 6ec94e12a2fc368541ee9e57e0d236630e4033a7